src/scrapeCrashes.ts

```diff
@@ -85,18 +85,16 @@ export async function scrapeCrashes(argv: any) {
 
     const availablePackages = await downloader.getOverview();
     // Remove any suspended and draft apps from the set of available packages as these aren't in use.
-    const publishedPackages = availablePackages.filter(p => p.status == "Published");
-    console.log("Details of the published packages for this account:")
-    console.log(publishedPackages)
+    const publishedPackages = availablePackages.filter(p => p.status === 'Published');
     const publishedPackageNames = publishedPackages.map(p => p.packageName);
-     let packageNamesToScrape = argv.packageName.split(',');
-    if (packageNamesToScrape == '*') {
```

Comment:
`==` is frowned upon, in this case it kind of worked, but only because `packageNamesToScrape = ['*']`, and `==` coerced it into `'*'`. It's more stable to use `.includes('*')`

Reply:
SGTM. I think your original implementation used the === test which didn't work, I'd simply experimented and  found that my change worked, albeit because of the coersion at run time. Thank you for improving the code.